spec/features/user/user_index_spec.rb

```diff
@@ -0,0 +1,21 @@
+user_index_spec.rb
```

Comment:
- [ ] Consider deleting this line, it makes running the tests fail. See the screenshot below 👇🏼:
![error](https://user-images.githubusercontent.com/45002654/141235869-d34fee9c-4d53-4307-a950-bfbf52e36fe1.png)